### PR TITLE
Link included assets and entries in QueryAPI.getEntries

### PR DESCRIPTION
**Pull request by baering**

### CODE REVIEW

File: packages/tux-adapter-contentful/src/query-api.ts [hunk 1]
```diff
@@ -18,8 +18,9 @@ class QueryApi {
   }
 
   async getEntries(params? : Object) {
-    const result = await this.client.get('/entries', { params }).then(result => result.data)
+    const result = await this.client.get('/entries?include=1', { params }).then(result => result.data)
```
Comment (thorsteinsson) [inline packages/tux-adapter-contentful/src/query-api.ts]: Should we have this as an option in the params? so if `params.include` do this functionality.

File: packages/tux-adapter-contentful/src/query-api.ts [hunk 2]
```diff
@@ -32,6 +33,37 @@ class QueryApi {
...
+
+    this.populateLinks(result.includes.Asset, linkMap)
+    this.populateLinks(result.includes.Entry, linkMap)
+
+    for (const item of result.items) {
+      const fieldNames = Object.keys(item.fields)
+      for (const fieldName of fieldNames) {
+        const field = item.fields[fieldName]
+        if (field instanceof Array) {
+          for (const childField of field) {
```
Comment (baering) [inline packages/tux-adapter-contentful/src/query-api.ts:57]: Behold, the three level for loop. #bestcodeihavewritten

File: packages/tux-adapter-contentful/src/query-api.ts (hunk 1 above)
Comment (eirikurn) [reply packages/tux-adapter-contentful/src/query-api.ts]: The api should have include=1 by default, making this redundant: https://www.contentful.com/developers/docs/concepts/links/#link-level

This library should always link includes as needed, so if someone passes { include: 10 }, it still links properly.

Comment (eirikurn) [reply packages/tux-adapter-contentful/src/query-api.ts]: Also, the link resolver should also "checkOverride". So if I edit an included model/asset, it surfaces correctly

**Pull request merged**